there is not .d.ts file in dist after build source

I used this lib works with typescript, when I import ts-pegjs lib use with 'import tspegjs from "ts-pegjs"', it got an error shows "Cannot find module 'ts-pegjs' or its corresponding type declarations".     Then I clone your lib source to build locally and found that not any .d.ts file in dist folder.   Is it possible to add some typing files to the lib dist?
<img width="1153" alt="截屏2023-05-14 11 20 11" src="https://github.com/metadevpro/ts-pegjs/assets/72691225/a632e7f2-f09f-4487-9596-14038098ac89">

--- a/package.json
+++ b/package.json
@@ -27,6 +27,7 @@
     "import": "./dist/tspegjs.mjs",
     "require": "./dist/tspegjs.js"
   },
+  "types": "./dist/tspegjs.d.ts",
   "files": [
     "LICENSE",
     "package.json",
@@ -55,6 +56,7 @@
     "rollup-plugin-add-shebang": "^0.3.1",
     "typescript": "^5.0.4",
     "vite": "^4.3.8",
+    "vite-plugin-dts": "^2.3.0",
     "vite-tsconfig-paths": "^4.2.0",
     "vitest": "^0.31.1"
   },

--- a/vite.config.ts
+++ b/vite.config.ts
@@ -1,6 +1,7 @@
 import viteTsconfigPaths from 'vite-tsconfig-paths';
 import rollupPluginShebang from 'rollup-plugin-add-shebang';
 import { defineConfig } from 'vitest/config';
+import dts from 'vite-plugin-dts';
 
 /**
  * The main configuration for Vite. This config includes
@@ -9,7 +10,10 @@ import { defineConfig } from 'vitest/config';
 export default defineConfig({
   plugins: [
     viteTsconfigPaths(),
-    rollupPluginShebang({ include: ['**/cli.js', '**/cli.mjs'], shebang: '#!/usr/bin/env node' })
+    rollupPluginShebang({ include: ['**/cli.js', '**/cli.mjs'], shebang: '#!/usr/bin/env node' }),
+    dts({
+      exclude: ["./src/cli.ts"]
+    }),
   ],
   base: './',
   build: {

